### PR TITLE
Added student projects

### PR DESCRIPTION
**Pull request by 1Niels**
added projects to gallery and to Geolocation Projects

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://1Niels.github.io/gci15.fossasia.org

File: index.html [hunk 1]
```diff
@@ -438,6 +438,21 @@ <h2 class="photostack-title"><a href="http://nolik2000.github.io." target="_blan
               <h3>Nastya Kostiv</h3>
             </figcaption>
           </figure>
+		<figure>
+         	<img src="https://cloud.githubusercontent.com/assets/9834624/12374282/d01df64a-bc64-11e5-8c5f-df8d9643a333.png" alt="img33" >
```
Comment (jigyasa-grover) [inline index.html:442]: Add image in the `img/gallery` and then access it here. Follow the format @1Niels

**Comment by SaptakS** [comment]
This branch has merge conflicts... Please see into it.. Thanks..

**Comment by jigyasa-grover** [comment]
[This](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) shall help @1Niels :)

**Comment by jigyasa-grover** [comment]
Updates @1Niels  ? Feel free to ask questions, if any.

**Comment by jigyasa-grover** [comment]
@1Niels Closing this.

**Pull request closed without merging**